tests/Unit/PointwiseFunctions/AnalyticData/CurvedWaveEquation/Test_ScalarWaveGr.cpp

```diff
@@ -210,6 +210,105 @@ void test_construct_from_options() noexcept {
     test_kerr(curved_wave_data_constructed_from_opts, bh_solution,
               flat_wave_solution);
   }
+  {  // test flat spacetime with wave profile = plane in 3D
+    const auto curved_wave_data_constructed_from_opts =
+        TestHelpers::test_creation<CurvedScalarWave::AnalyticData::ScalarWaveGr<
+            ScalarWave::Solutions::PlaneWave<3>, gr::Solutions::Minkowski<3>>>(
+            "Background:\n"
+            "ScalarField:\n"
+            "  WaveVector: [1.0, 1.0, 1.0]\n"
+            "  Center: [0.0, 0.0, 0.0]\n"
+            "  Profile:\n"
+            "    Gaussian:\n"
+            "      Amplitude: 11.\n"
+            "      Width: 1.4\n"
+            "      Center: 0.3");
+
+    // construct internals of data constructed from options
+    const gr::Solutions::Minkowski<3> flat_space_solution{};
+    const ScalarWave::Solutions::PlaneWave<3> flat_wave_solution{
```

Comment:
This isn't used.  Can you move it below where you comment that you can't copy it?  Same for the other tests.

Reply:
removed them in fact, there's no use constructing these objects.